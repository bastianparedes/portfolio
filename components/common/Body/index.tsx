import React from 'react'
import Footer from '../Footer'
import styles from './styles.module.scss'

interface props {
  children?: React.ReactNode
}

const Body = ({ children }: props): JSX.Element => {
  return (
    <>
      <div className={styles.body}>
        {children}
      </div>
      <Footer />
    </>
  )
}

export default Body
