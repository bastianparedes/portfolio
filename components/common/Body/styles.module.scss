.body {
  min-width: 100%;
  min-height: calc(100vh - 3rem);
  margin: 0;
  border: none;
  padding: 0;
}
