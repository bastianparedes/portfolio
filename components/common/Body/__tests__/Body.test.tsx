import React from 'react';
import Body from '..';
import { render } from '@testing-library/react';

jest.mock('../../Footer');

describe('<Body />', () => {
  it('should render', () => {
    const { container } = render(<Body />);
    expect(container).toMatchSnapshot();
  });
});
