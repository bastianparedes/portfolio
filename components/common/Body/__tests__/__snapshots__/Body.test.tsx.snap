// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Body /> should render 1`] = `
<div>
  <div
    class="body"
  />
</div>
`;
